src/app/Dashboard/Charts/jfr/JFRMetricsChartCard.tsx

```diff
@@ -321,8 +323,8 @@ export const JFRMetricsChartCardDescriptor: DashboardCardDescriptor = {
     {
       name: 'CHART_CARD.PROP_CONTROLS.THEME.NAME',
       key: 'theme',
-      values: ['light', 'dark'],
-      defaultValue: 'light',
+      values: Object.values(ThemeType),
+      defaultValue: getFromLocalStorage('THEME', ThemeType.LIGHT),
```

Comment:
I think this is only intialized once so if the user changes preferences in between, it wont get updated. Maybe just remove this config and add a subscription to theme in JFR charts?